wandb/apis/public.py

```diff
@@ -843,6 +851,70 @@ def __repr__(self):
         return "<Runs {}/{}>".format(self.entity, self.project)
 
 
+class QueuedJob(Attrs):
+    def __init__(self, client, entity, project, queue, run_queue_item_id, attrs={}):
+        super(QueuedJob, self).__init__(dict(attrs))
+        self.client = client
+        self._entity = entity
+        self.project = project
+        self._queue = queue
+        self._run_queue_item_id = run_queue_item_id
+        self._run_id = None
+
+    def wait_til_running(self):
```

Comment:
where does this get called? also fn name is kinda confusing not entirely sure what this does

Reply:
Called by user in their own process.